tests/test_matrix_utils_connect.py

```
@@ -5,11 +5,23 @@
+from unittest.mock import AsyncMock, MagicMock, Mock, patch
 
 import pytest
 
-from mmrelay.matrix_utils import MissingMatrixRoomsError, connect_matrix
+from mmrelay.matrix_utils import MatrixAuthInfo, MissingMatrixRoomsError, connect_matrix
+
+
+class _ImmediateAwaitable:
+    """Awaitable that resolves immediately without creating coroutine objects."""
+
+    def __init__(self, value=None):
```

Ruff (0.15.10)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)